app/src/main/java/org/kiwix/kiwixmobile/nav/destination/library/OnlineLibraryFragment.kt

```diff
@@ -247,17 +247,21 @@ class OnlineLibraryFragment : BaseFragment(), FragmentActivityExtensions {
         })
         return
       }
-      else -> availableSpaceCalculator.hasAvailableSpaceFor(item,
-        { downloadFile(item.book) },
-        {
-          libraryList.snack(
-            getString(R.string.download_no_space) +
-              "\n" + getString(R.string.space_available) + " " +
-              it,
-            R.string.download_change_storage,
-            ::showStorageSelectDialog
-          )
-        })
+      else -> if (sharedPreferenceUtil.showStorageOption) {
+          showStorageSelectDialog()
+        } else {
+          availableSpaceCalculator.hasAvailableSpaceFor(item,
+            { downloadFile(item.book) },
+            {
+              libraryList.snack(
+                getString(R.string.download_no_space) +
+                  "\n" + getString(R.string.space_available) + " " +
+                  it,
+                R.string.download_change_storage,
+                ::showStorageSelectDialog
```

Comment:
The code is not formatted use `option` + `enter` in mac or `alt` + `enter` in windows or Linux to fix this.

Reply:
Nope, you're incorrect, the default binding is: `Ctrl/Cmd + Alt + L`

Reply:
@s-ayush2903 What I am saying is let the android studio fix the format issue.

Reply:
I also told the same : ) 
What you told, doesn't fix formatting, instead it does some minor fix in the code(which isn't related to formatting)